applicationRow: Use HdyActionRow

BrainBlasted opened the pull request:
HdyActionRow gives us title and subtitle styling for
list rows that's consistent with other GNOME apps.

The previous styling used italics for subtitles,
which is general discouraged. See: https://developer.gnome.org/hig/guidelines/typography.html

BrainBlasted committed:
applicationRow: Use HdyActionRow

HdyActionRow gives us title and subtitle styling for
list rows that's consistent with other GNOME apps.

The previous styling used italics for subtitles,
which is general discouraged. See: https://developer.gnome.org/hig/guidelines/typography.html

tchx84 commented:
I see a lot of red in that diff, thanks Chris :) Ill test this during the weekend.

BrainBlasted committed:
applicationRow: Adjust icon sizing

App icons are built to display at multiples of 16px.
They do not scale properly to 40px. This commit
changes the icon size to 48px, which makes icons sharper.

BrainBlasted commented:
Added a commit to adjust icon size.

tchx84 commented:
This looks pretty good! Thanks!
